swagger-console.yml

```
@@ -2867,6 +2867,315 @@ paths:
       tags:
         - Logging
 
+  /kms/status:
+    get:
+      summary: KMS status
+      operationId: KMSStatus
+      responses:
+        200:
+          description: A successful response.
+          schema:
+            $ref: "#/definitions/kmsStatusResponse"
+        default:
+          description: Generic error response.
+          schema:
+            $ref: "#/definitions/error"
+      tags:
+        - KMS
+  /kms/key/create:
+    post:
+      summary: KMS create key
+      operationId: KMSCreateKey
+      parameters:
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/kmsCreateKeyRequest"
+      responses:
+        201:
+          description: A successful response.
+        default:
+          description: Generic error response.
+          schema:
+            $ref: "#/definitions/error"
+      tags:
+        - KMS
+  /kms/key/import:
+    post:
+      summary: KMS import key
+      operationId: KMSImportKey
+      parameters:
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/kmsImportKeyRequest"
+      responses:
+        201:
+          description: A successful response.
+        default:
+          description: Generic error response.
+          schema:
+            $ref: "#/definitions/error"
+      tags:
+        - KMS
+  /kms/key/list:
+    get:
+      summary: KMS list keys
+      operationId: KMSListKeys
+      parameters:
+        - name: pattern
+          description: pattern to retrieve keys
+          in: query
+          required: true
+          type: string
+      responses:
+        200:
+          description: A successful response.
+          schema:
+            $ref: "#/definitions/kmsListKeysResponse"
+        default:
+          description: Generic error response.
+          schema:
+            $ref: "#/definitions/error"
+      tags:
+        - KMS
+  /kms/key/status:
```

```suggestion
  /kms/key/{id}:
```
Passing the key id should return information about the key including the status